rules.md

```diff
@@ -122,17 +99,13 @@ All Bounties start as a Concept that is [proposed](#31-proposing-concepts) to th
 
 Concepts are proposals for some work that the Incubator should fund as a Bounty, that provide enough detail about the value proposition and requirements of the work to enable Contributors to complete it.
 
-[Admins](#4-admins) create a Bounty for each Concept they Accept that they then Own, meaning they have the rights to manage and earn commission from its delivery.
-
-Admins can [Accept](#435-processing-task-claims) any Concept that meets our [Concept Acceptance Criteria](#411-concept-acceptance-criteria) on a first-come first-serve basis.
-
-Once accepted, the user who created the Concept is automatically assigned as a [Stakeholder](#25-bounty-stakeholders) of the corresponding bounty.
+[Strategists](#46-strategists) create a Bounty for each Concept they accept that they then own, meaning they have the rights to manage and earn commission from its delivery.  They [accept](#434-processing-task-claims) Concepts that meet our [Concept Acceptance Criteria](#411-concept-acceptance-criteria) on a first-come first-serve basis.  Once accepted, the user who created the Concept is assigned as a [Stakeholder](#25-bounty-stakeholders) of the corresponding bounty.
 
 ## 2.2 Bounty Types
 
-When a Bounty is created from a [Concept](#21-concepts) by an [Admins](#4-admins), they assign one of four types for the new Bounty that determine how work will be structured:
+When a Bounty is created from a [Concept](#21-concepts) by a [Strategist](#46-strategists), they assign one of four types for the new Bounty that determine how work will be structured:
 
-- **Projects** develop products such as a decentralized app built on Dash (dApps), tools for the community or improvements to the Dash Protocol itself.
+- **Projects** develop products such as a decentralized apps built on Dash (dApps), tools for the community or improvements to the Dash Protocol itself.
```

Comment:
'(dApps)' should come after 'decentralized apps'

Reply:
Fixed in https://github.com/dashincubator/dash-incubator-rules/pull/11/commits/289a9577324c2a6c85bb4101145ffea64c8d231c

Reply:
Fixed typo: https://github.com/dashincubator/dash-incubator-rules/pull/11/commits/410ab24f42233b3a83dc454a162c2a4a50b6e793